lib/kitchen/directions/bake_numbered_table/v1.rb

```diff
@@ -2,18 +2,19 @@ module Kitchen::Directions::BakeNumberedTable
   class V1
 
     def bake(table:, number:)
-      table.wrap(%(<div class="os-table">))
-
       table_label = "#{I18n.t(:table_label)} #{number}"
       table.document.pantry(name: :link_text).store table_label, label: table.id
 
       if table.top_titled?
+        table.wrap(%(<div class="os-table os-top-titled-container">))
```

Comment:
instead of having the `.wrap` call in two places, you can leave it where it is above and then do `table.parent.add_class("os-top-titled-container")`